spec.yaml

```diff
@@ -108,6 +108,19 @@ paths:
               type: string
           schema:
             $ref: "#/definitions/Error"
+  /mfa-enforcing:
```

Comment:
Maybe this endpoint could be better named `/mfa-capable`? What I mean is that an `mfa-capable` OCM provider does not necessarily enforce MFA in all shares, but only on those where the `mfa-enforced` permission is set.

Also, I think it can be simplified with just an empty `200` response, as an implementation that does not have such capability would respond with a `404`.

Reply:
Good point!